java/osgi/OsgiWrapper.java

```diff
@@ -122,6 +126,8 @@ public Integer call() throws Exception {
     analyzer.setProperty(Analyzer.BUNDLE_DOCURL, bundleDocUrl);
     analyzer.setProperty(Analyzer.BUNDLE_LICENSE, bundleLicense);
     analyzer.setProperty(Analyzer.REMOVEHEADERS, REMOVEHEADERS);
+    analyzer.setProperty(MULTI_RELEASE_MANIFEST_PROPERTY, "true");
+    analyzer.setProperty(FIXUP_MESSAGES_PROPERTY, FIXUP_MESSAGES_IGNORE_MRJAR);
```

Comment:
Adjusts BND arguments in the following manner

- `Automatic-Module-Name` is removed
- `Multi-Release: true` is added
- `-fixupmessages` is passed to suppress a BND validation failure when classes reside in `META-INF/versions`, which is legal for MRJARs.

These same changes are applied to the BND tools invocation in Error Prone and Guava.

Reply:
Does bnd validation not support multi-release jars?

Reply:
@Sineaggi It just identifies `META-INF/versions` as "invalid classes," which of course they aren't. I'm happy to dig in as I find it to be a bit strange too.

Reply:
Ultimately Protobuf is not being "built" or re-exported with different class hierarchy at different versions; only the module identifier remains as a compiled class within that root. So I am figuring that since `META-INF` and `module-info` are not valid class identifiers _outside of_ MRJARs and modular JARs, maybe that's where Bnd is choking.

But you're right, I'm not sure, that's a guess.

Reply:
(Strange, because bnd _is_ capable of generating its own `module-info.java` definitions; it is a feature described in their [JPMS section](https://bnd.bndtools.org/chapters/330-jpms.html). MRJARs were created mostly to allow the safe transition to JDK 9+, so I would imagine that if they support JPMS stuff they should support MRJARs.)